expression/builtin_time.go

```diff
@@ -1760,7 +1764,77 @@ type builtinConvertTzSig struct {
 // eval evals a builtinConvertTzSig.
 // See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_convert-tz
 func (b *builtinConvertTzSig) eval(row []types.Datum) (d types.Datum, err error) {
-	return d, errFunctionNotExists.GenByArgs("CONVERT_TZ")
+	args, err := b.evalArgs(row)
+	if err != nil {
+		return
+	}
+
+	if args[0].IsNull() || args[1].IsNull() || args[2].IsNull() {
+		return
+	}
+
+	sc := b.ctx.GetSessionVars().StmtCtx
+	fsp := types.DateFSP(args[0].GetString())
+	if fsp > types.MaxFsp {
+		fsp = types.MaxFsp
+	}
+
+	date, err := convertToTimeWithFsp(sc, args[0], mysql.TypeDatetime, fsp)
+	if err != nil {
+		return
+	}
+
+	if date.IsNull() {
+		return
+	}
+
+	mysqlTime := date.GetMysqlTime()
```

Comment:
`mysqlTime` is ambiguous,
can we rename `date` to `arg0`, rename `mysqlTime` to `dt` which is used in MySQL manual?